### ISSUE TITLE
Scale-up-animation, cascade-animation stopped working after update

### ISSUE THREAD

**Issue by agozie** [PolymerElements/neon-animation#40]
The scale-up-animation worked until I did bower update a few days ago. Now only the fade-out-animation work. Replacing the entry-animation with other animations work. Also cascade-animation stopped working after that update. The scale-up-animation used in the dropdown example stopped working too. What am I missing?

 ```javascript 
<paper-dialog id="lDiag" entry-animation="scale-up-animation" exit-animation="fade-out-animation">
            <h2>Hi</h2>
                <p>Hello</p>
            <div class="buttons">
                <paper-button dialog-dismiss>Cancel</paper-button>
                <paper-button dialog-confirm on-tap="showTDetails">OK</paper-button>
            </div>
   </paper-dialog>
 ```

**Comment by timeu** [comment]
I think this is related to this change: 

https://github.com/PolymerElements/neon-animation/commit/cf9f1f44298b3721e8d498904e583510e4a8d633#diff-a15586f46b1093a65de0217a267ef354

The default `scaleProperty` should not be `scale(1,1)` but the `scale(0)`

**Comment by agozie** [comment]
scale(0) solved the issue thanks!

**Comment by cprecioso** [comment]
Sorry, my fault, didn't catch it because my project doesn't `scale-up` on both axes.

**Comment by morethanreal** [comment]
Fixed via #41.
